mobsfscan/rules/patterns/ios/swift/best_practices.yaml

```diff
@@ -80,7 +84,7 @@
     reference: https://github.com/MobSF/owasp-mstg/blob/master/Document/0x06d-Testing-Data-Storage.md#finding-sensitive-data-in-the-keyboard-cache-mstg-storage-5
 - id: ios_uiwebview_safari_view
   message: >-
-    This app uses SFSafariViewController or UIWebView. It 
+    This app uses SFSafariViewController or UIWebView. It
```

Comment:
Then isn't this is incorrectly positive as a best_practice?

Reply:
Yeah, you are right. I will move this shortly.